src/sidebar/components/NotebookView.tsx

```diff
@@ -133,8 +134,14 @@ function NotebookView({ loadAnnotationsService, streamer }: NotebookViewProps) {
           {groupName}
         </h1>
       </header>
-      <div className="absolute w-full z-5 left-0 top-7">
-        <div className="container flex flex-row-reverse">
+      <div className="absolute w-full z-5 left-0 top-8">
+        <div
+          className={classnames(
+            'container flex flex-row-reverse relative',
+            // Compensate container's right padding
```

Comment:
```suggestion
            // Compensate for container's right padding
```

Reply:
Where is that right padding set?

Reply:
[Here](https://github.com/hypothesis/client/blob/23c83fef1c83ed431896dfc6b9f059bbbd1471e3/tailwind.config.mjs#L44-L61), but I'm glad you asked, because this needs to be set differently per breakpoint.

I have to admit the overall styling here is a bit hacky, but it's the only way to make it behave as if it was position fixed, ensureing it is not "lost" when scrolling down", but render relative to the main content block, not the viewport.

Reply:
I made a few changes and I think this looks reasonably well: Aligned with the content in desktop resolutions, and aligned with the close button in mobile ones.

https://github.com/hypothesis/client/assets/2719332/6a749632-c0e5-4de4-a059-e936b24efcd3

However, there's an intermediary range of sizes where it does not keep alignment with the content. I'm checking that.

Reply:
OK, that looks good.